apps.yml

```diff
@@ -4058,3 +4058,15 @@ apps:
     url: https://app.learnerbly.com/auth-init/?customProvider=9bd4f3ef
     vanity_url:
     - /learnerbly
+- application:
+    authorized_groups:
+    - mozilliansorg_web-sre-papertrail-access
+    authorized_users: []
+    client_id: QSbhAzqUlqCSWt6iAV45um5DDGdhhDTR
+    display: true
+    logo: papertrail-logo.png
+    name: Papertrail Web SRE
+    op: auth0
+    url: https://auth.mozilla.auth0.com/samlp/QSbhAzqUlqCSWt6iAV45um5DDGdhhDTR
```

Comment:
This url is a little different than what I would generally expect.  I'd expect it to be the papertrail domain with whatever path gets you to a nice login page.

Reply:
This would be a normal URL for a SAML integration. the `url` field should be the URL that when you browse to it, it continues a redirect through to have you logged into the site.

Reply:
Afaik, this URL is consistent with many application already in place.  In my testing, I also discovered, the Solarwinds Unified login page doesn't work if you do not enter the aforementioned URL at least once.  On first authentication, it creates a local user based on role/group mapping from the SAML response.  If you do not have the proper roles in your SAML response, you are bounced back to the Solarwinds Login page.

I'll also note here that each Solarwinds org will need its own auth0 SAML application.  If your email address is assosicated with multiple orgs, like mine is, Solarwinds will SAML authenticate you for each one that has SAML enabled.  If you are also a member of a non-SAML enabled org, you will be instructed to log into Solarwinds with a standard Solarwinds account in order to authenticate you to any remaining orgs.